gammapy/cube/tests/test_make.py

```diff
@@ -70,7 +70,7 @@ def geom(ebounds):
             "counts": 34366,
             "exposure": 5.971096e11,
```

Comment:
This is definitely not a part of this PR, but I see that the exposure values also vary a lot depending on the number of (true) energy bins. Perhaps changing from the `evaluate()` to ` _trapz_loglog` integral might help in this case as well?